.gitignore

```diff
@@ -140,7 +140,11 @@ cython_debug/
 # asv
 Django/
 html/
-results/
+results/*
```

Comment:
There was some previous discussion about removing the `results` folder from here. That seems simpler than keeping a list of exceptions? 

If folk want to ignore results on their own machine they can configure this locally. See https://github.com/django/django/blob/8d160f154f0240a423e83ffe0690e472f837373c/.gitignore#L2

Reply:
I have removed the list of exceptions and ignored the `results` folder, in Github actions the results will be added forcefully.

Reply:
OK, we should add some guidance on how to do this in the readme, so that if folk want to contribute their own results there's documentation on how to do that.